### ISSUE TITLE
bug(MatSort/MatTooltip): Sort headers keyboard focus indicator not working when the header has a MatTooltip

### ISSUE THREAD

**Issue by Warwizza** [angular/components#19218]
#### Reproduction
Use StackBlitz to reproduce your issue: https://stackblitz.com/edit/angular-bvhoxg?file=src/app/table-sorting-example.html

Steps to reproduce:
1. Tab through the table columns and notice the weight header column does not have a keyboard focussed indicator. 
2. If you remove the tooltip the column the indicator works as expected.
  
#### Expected Behavior
What behavior were you expecting to see?
All columns should get the keyboard focussed indicator. Not only the columns without a tooltip.

#### Actual Behavior
What behavior did you actually see?
The header with the tooltip did not get the keyboard focussed indicator.

#### Environment
- Angular: 9+
- CDK/Material: 9+
- Browser(s): FF 75, Chrome 81 and Edge 81
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

First issue: https://github.com/angular/components/issues/17716
